data_generation.py

```diff
@@ -1,17 +1,17 @@
-import csv
+"""Here it is going to generate a sample"""
 import random
 import json
 
 
 if __name__ == "__main__":
-    number_of_sample = 20
-    C = 760000
+    NUMBER_OF_SAMPLE = 20
+    C = 760_000
 
-    with open("dataset.csv", "w") as file:
-        line = "force,torque\n"
-        file.write(line)
-        for i in range(number_of_sample):
-            P = C * (random.randint(10, 200) / 100) # Parameters P will have randomly from 10% to 200% of C value
+    with open("dataset.csv", "w", encoding="utf-8") as file:
+        LINE = "force,torque\n"
+        file.write(LINE)
+        for i in range(NUMBER_OF_SAMPLE):
+            P = C * (random.randint(10, 200) / 100)
             torque = 3_000 * (random.randint(10, 200) / 100)
             line = f"{P},{torque}\n"
             file.write(line)
```

Comment:
This is a nice snipped of how we would perform the task in a low-level language like c. However, python gives us many handy tools that simplify daily workflow, so that you don't have to program such typical things from scratch.
Please consider the usage of numpy and pandas:

```
import numpy as np
import pandas as pd

P = np.random.random(lower_bound1, higher_bound1, number_of_samples)
torque = np.random.random(lower_bound2, higher_bound2, number_of_samples)

data = pd.DataFrame(
    {
        "P": P,
        "torque": torque
     }
)

data.to_csv("dataset.csv")
```

I know that it might not look like a simplification at the first sight but this is how we would solve a similar problem in our project.